content/help/ops/setup/webhook/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Dynamic Admission Webhooks
+description: Describes Istio's use of Kubernetes webhooks and the related issues that can arise.
+weight: 10
+---
+
+This page assumes you're familiar with the [Kubernetes mutating and
```

Comment:
Can we give here a one paragraph explanation, and then give a single link for them to get more info?